.gitignore

```diff
@@ -7,9 +7,6 @@
 # Log files
 *.log
 
-# Coverage
```

Comment:
Question (@CBenoit): Is coverage using `docs` directory for some reason, or is this an accidental .gitignore item? 🤔

Reply:
Not accidental, this folder is created by the coverage task: https://github.com/Devolutions/IronRDP/blob/9470d2a7c2835e634f0a907db43d1f7165faccb3/xtask/src/cov.rs#L204
That's why there is a "Coverage" comment I think :slightly_smiling_face: 

Try to run the task and see if we can ignore `docs/coverage` only instead? Probably some other files are placed into the `docs/` folder so I'm unsure.
Alternatively, use something else than `docs/` for the coverage, and ignore that instead.

Reply:
I have tested `grcov` command, the docs are only placed inside "docs/coverage", so I added that folder to `.gitignore` instead of the whole `docs`.